.circleci/config.yml

```diff
@@ -48,6 +48,11 @@ jobs:
       - checkout
       - install-golangci-lint
       - *restore_cache
+      - run:          
+          command: |
+            make assets
+            git diff
```

Comment:
I actually found a cool way to achieve this. Just use `git diff --exit-code`. It will exit with 1 if there are differences. Credit to @hanzei for finding this trick.